lib/smart_proxy_pulp_plugin/pulp_disk.rb

```diff
@@ -0,0 +1,21 @@
+module PulpProxy
+  class PulpDisk
+    def initialize(path = PulpProxy::Plugin.settings.pulp_dir)
+      @path = path
+    end
+
+    def df_h
+      `df -h #{@path}`.split("\n").last
```

Comment:
This will work, but it can segfault on non-Linux platforms (like BSD and stuff)...

```
require 'ffi'

class Statvfs < FFI::Struct
  layout(
    :f_bsize, :ulong,
    :f_frsize, :ulong,
    :f_blocks, :ulong,
    :f_bfree, :ulong,
    :f_bavail, :ulong,
    :f_files, :ulong,
    :f_ffree, :ulong,
    :f_favail, :ulong,
    :f_fsid, :ulong,
    :f_flag, :ulong,
    :f_namemax, :ulong,
    :f_ftype, :ulong,
    :f_basetype, [:char, 16],
    :f_str, [:char, 16]
  )
end

module MyLib
  extend FFI::Library
  ffi_lib 'c'
  attach_function(:statvfs, [:string, :pointer], :int)
end

struct = Statvfs.new
MyLib.statvfs("/var", struct)
struct[:f_bfree] * struct[:f_bsize]

=> 28866289664
```